components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/GatewayEnvironmentProtocolURIDTO.java

```diff
@@ -0,0 +1,103 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+
+import javax.xml.bind.annotation.*;
+import org.wso2.carbon.apimgt.rest.api.common.annotations.Scope;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import javax.validation.Valid;
+
+
+
+public class GatewayEnvironmentProtocolURIDTO   {
+  
+    private String protocol = null;
+    private String endpointURI = null;
+
+  /**
+   **/
+  public GatewayEnvironmentProtocolURIDTO protocol(String protocol) {
+    this.protocol = protocol;
+    return this;
+  }
+
+  
+  @ApiModelProperty(example = "default", required = true, value = "")
+  @JsonProperty("protocol")
+  @NotNull
+  public String getProtocol() {
+    return protocol;
+  }
+  public void setProtocol(String protocol) {
+    this.protocol = protocol;
+  }
+
+  /**
+   **/
+  public GatewayEnvironmentProtocolURIDTO endpointURI(String endpointURI) {
+    this.endpointURI = endpointURI;
+    return this;
+  }
+
+  
+  @ApiModelProperty(example = "default", required = true, value = "")
+  @JsonProperty("endpointURI")
+  @NotNull
+  public String getEndpointURI() {
+    return endpointURI;
+  }
+  public void setEndpointURI(String endpointURI) {
+    this.endpointURI = endpointURI;
+  }
+
+
+  @Override
+  public boolean equals(java.lang.Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    GatewayEnvironmentProtocolURIDTO gatewayEnvironmentProtocolURI = (GatewayEnvironmentProtocolURIDTO) o;
+    return Objects.equals(protocol, gatewayEnvironmentProtocolURI.protocol) &&
+        Objects.equals(endpointURI, gatewayEnvironmentProtocolURI.endpointURI);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(protocol, endpointURI);
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("class GatewayEnvironmentProtocolURIDTO {\n");
+    
```

Comment:
remove new line

Reply:
This file is an auto-generated DTO file. Hence we cannot change anything on this file.